--- diag-ether/eepro100-diag.c.org	2004-02-22 21:31:11.000000000 +0000
+++ diag-ether/eepro100-diag.c	2004-02-22 21:33:36.000000000 +0000
@@ -34,37 +34,36 @@
 "  For details and other options see http://www.scyld.com/diag/index.html\n";
 
 static const char long_usage_msg[] =
-"Usage: %s [-aDfrRvVw] [-AF <speed+duplex>] [-#<BoardNum>]
-
-   Show the internal state of a network adapter.
-
-   The common usage is
-      diag -aem
-
- Frequently used options are
-   -a  --show_all_registers	Print all registers.
-   -e  --show-eeprom	Dump EEPROM contents, \"-ee\" shows the details.
-   -m  --show_mii		Print the MII transceiver state
-					Using -mm monitors the link.
-   -f  --force		Perform operation, even on a running NIC.
-
- To operate on a single NIC, or one that hasn't been automatically found:
-   -#  --card_num	INDEX	Operate on the specified card index.
-   -p  --port-base	IOADDR	Assume an adapter at the specified I/O address.
-   -t  --chip-type	TYPE	Specify adapter type (with '-p'), use '-1' to list.
-
- To change the persistent EEPROM settings
-   -G  --parameters PARMS	Set adapter-specific parameters.
-   -H  --new-hwaddr 01:23:45:67:89:ab
-        Set a new hardware station address.  Typically disabled for safety.
-   -w  --write-EEPROM	 Actually write the new settings into the EEPROM.
-
-   -D  --debug
-   -v  --verbose	Report each action taken.
-   -V  --version	Emit version information.
-
-   -A  --advertise <speed|setting>  (See the mii-diag manual page.)
-";
+"Usage: %s [-aDfrRvVw] [-AF <speed+duplex>] [-#<BoardNum>]\n"
+"\n"
+"   Show the internal state of a network adapter.\n"
+"\n"
+"   The common usage is\n"
+"      diag -aem\n"
+"\n"
+" Frequently used options are\n"
+"   -a  --show_all_registers	Print all registers.\n"
+"   -e  --show-eeprom	Dump EEPROM contents, \"-ee\" shows the details.\n"
+"   -m  --show_mii		Print the MII transceiver state\n"
+"					Using -mm monitors the link.\n"
+"   -f  --force		Perform operation, even on a running NIC.\n"
+"\n"
+" To operate on a single NIC, or one that hasn't been automatically found:\n"
+"   -#  --card_num	INDEX	Operate on the specified card index.\n"
+"   -p  --port-base	IOADDR	Assume an adapter at the specified I/O address.\n"
+"   -t  --chip-type	TYPE	Specify adapter type (with '-p'), use '-1' to list.\n"
+"\n"
+" To change the persistent EEPROM settings\n"
+"   -G  --parameters PARMS	Set adapter-specific parameters.\n"
+"   -H  --new-hwaddr 01:23:45:67:89:ab\n"
+"        Set a new hardware station address.  Typically disabled for safety.\n"
+"   -w  --write-EEPROM	 Actually write the new settings into the EEPROM.\n"
+"\n"
+"   -D  --debug\n"
+"   -v  --verbose	Report each action taken.\n"
+"   -V  --version	Emit version information.\n"
+"\n"
+"   -A  --advertise <speed|setting>  (See the mii-diag manual page.)\n\n";
 
 #if ! defined(__OPTIMIZE__)
 #warning  You must compile this program with the correct options!
