--- diag-ether/ether-wake.c.org	2004-02-22 22:32:08.000000000 +0000
+++ diag-ether/ether-wake.c	2004-02-22 22:32:52.000000000 +0000
@@ -6,30 +6,30 @@
 "usage: ether-wake [-i <ifname>] [-p aa:bb:cc:dd[:ee:ff]] 00:11:22:33:44:55\n"
 "   Use '-u' to see the complete set of options.\n";
 static char usage_msg[] =
-"usage: ether-wake [-i <ifname>] [-p aa:bb:cc:dd[:ee:ff]] 00:11:22:33:44:55
-
-	This program generates and transmits a Wake-On-LAN (WOL)
-	\"Magic Packet\", used for restarting machines that have been
-	soft-powered-down (ACPI D3-warm state).
-	It currently generates the standard AMD Magic Packet format, with
-	an optional password appended.
-
-	The single required parameter is the Ethernet MAC (station) address
-	of the machine to wake or a host ID with known NSS 'ethers' entry.
-	The MAC address may be found with the 'arp' program while the target
-	machine is awake.
-
-	Options:
-		-b	Send wake-up packet to the broadcast address.
-		-D	Increase the debug level.
-		-i ifname	Use interface IFNAME instead of the default 'eth0'.
-		-p <pw>		Append the four or six byte password PW to the packet.
-					A password is only required for a few adapter types.
-					The password may be specified in ethernet hex format
-					or dotted decimal (Internet address)
-		-p 00:22:44:66:88:aa
-		-p 192.168.1.1
-";
+"usage: ether-wake [-i <ifname>] [-p aa:bb:cc:dd[:ee:ff]] 00:11:22:33:44:55\n"
+"\n"
+"	This program generates and transmits a Wake-On-LAN (WOL)\n"
+"	\"Magic Packet\", used for restarting machines that have been\n"
+"	soft-powered-down (ACPI D3-warm state).\n"
+"	It currently generates the standard AMD Magic Packet format, with\n"
+"	an optional password appended.\n"
+"\n"
+"	The single required parameter is the Ethernet MAC (station) address\n"
+"	of the machine to wake or a host ID with known NSS 'ethers' entry.\n"
+"	The MAC address may be found with the 'arp' program while the target\n"
+"	machine is awake.\n"
+"\n"
+"	Options:\n"
+"		-b	Send wake-up packet to the broadcast address.\n"
+"		-D	Increase the debug level.\n"
+"		-i ifname	Use interface IFNAME instead of the default 'eth0'.\n"
+"		-p <pw>		Append the four or six byte password PW to the packet.\n"
+"					A password is only required for a few adapter types.\n"
+"					The password may be specified in ethernet hex format\n"
+"					or dotted decimal (Internet address)\n"
+"		-p 00:22:44:66:88:aa\n"
+"		-p 192.168.1.1\n"
+"\n";
 
 /*
 	This program generates and transmits a Wake-On-LAN (WOL) "Magic Packet",
